src/main/java/yevhen/bookstore/repository/impl/BookRepositoryImpl.java

```diff
@@ -0,0 +1,19 @@
+package yevhen.bookstore.repository.impl;
+
+import java.util.List;
+import org.springframework.stereotype.Repository;
+import yevhen.bookstore.model.Book;
+import yevhen.bookstore.repository.BookRepository;
+
+@Repository
+public class BookRepositoryImpl implements BookRepository {
+    @Override
+    public Book save(Book book) {
+        return null;
```

Comment:
i'm not sure but i implemented a functional here. Maybe I was wrong

Reply:
yeap! Let’s add realization;)